lab-kyle/app/html/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+  <head>
+    <link rel="stylesheet" type="text/css" href="bundle.css">
+    <script src="bundle.js"></script>
+  </head>
+  <body>
+    <main ng-controller="GameController as gameCtrl">
+      <form novalidate name="gameForm">
+        <select ng-model="gameCtrl.gameForm.direction">
```

Comment:
I don't see a need for the gameForm object. You could remove it. See comment in GameController as well.